force-app/main/default/classes/DMLOperations.cls

```diff
@@ -213,8 +320,25 @@ public with sharing class DMLOperations {
      * @param numOfCases The number of cases to create.
      */
     public static void createAndDeleteCases(String accountId, Integer numOfCases) {
-        // Insert the cases
+    List<Case> newCases = new List<Case>();
+
 
-        // Delete the cases
+for(Integer i = 0; i < numOfCases; i++){
+    
+    Case newCase = new Case();
+    newCase.Subject = 'Find me ice cream';
+    newCase.AccountId = accountId;
+    newCases.add(newCase);
+    }
+    insert newCases;
+		
+List<Case> casesToDelete = new List<Case>();
+
+for(Case eachCase : newCases){
+    ID caseIdToDelete = eachCase.Id;
+    casesToDelete.add(eachCase);
```

Comment:
The caseIdToDelete variable is assigned the value of the eachCase.Id but never used.
```suggestion
    Id caseIdToDelete = eachCase.Id;
    casesToDelete.add(eachCase);
```